sklearn/naive_bayes.py

```diff
@@ -154,9 +164,17 @@ class GaussianNB(BaseNB):
     [1]
     """
 
-    def __init__(self, priors=None):
+    def __init__(self, classes=None, priors=None):
+        self.classes = classes
         self.priors = priors
 
+        # initialize additional parameter:
+        self.classes_ = None
+
+        # initialize a flag which checks whether fit or partial fit was called
+        # once or not
+        self.fit_called = False
```

Comment:
using this to check first call to fit in place of _check_partial_fit_first_call. thoughts?